android/settings.gradle

```diff
@@ -1,3 +1,5 @@
+apply from: "dependencies.gradle"
+
 rootProject.name = 'reactNativeApp'
 include ':react-native-svg'
 project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
```

Comment:
should have
```
include ':react-native-native-modules'
project(':react-native-native-modules').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-native-modules/android')
```
here

Reply:
We do not want to distribute the packages to NPM, we distribute them to our Maven repository. Is there any technical reason why it should be in NPM?

Reply:
It's already in NPM.

Reply:
I think it is already published to npm `https://www.npmjs.com/package/react-native-native-modules`

ios has made it available like this, so it would be the best way to just do `yarn add react-native-native-modules` and react-native link react-native-native-modules` and it works on both platforms

Reply:
When it is in Maven repository and the dependency is listed in build.gradle, you do not need to call yarn at all on Android.

Reply:
I will approve from my side, we can always change it after it is merged